resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -706,6 +706,22 @@ private[spark] object Config extends Logging {
       .stringConf
       .createOptional
 
+  val KUBERNETES_FILE_UPLOADED_FAILS =
+    ConfigBuilder("spark.kubernetes.uploaded.files")
+      .internal()
+      .doc("Remember all uploaded uri by spark client, used to delete uris when app finished.")
+      .version("3.1.2-internal")
```

Comment:
Thank you for making a PR, but Apache Spark codebase needs a valid Spark version info, @thousandhu .

Reply:
I've changed version to 3.5.0.
BTW, this is a internal config, we don't want user to set it. So the config is set as .internal().